app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -20,9 +20,17 @@ class ParkingViewModel(
     private val parkingRepository: ParkingRepository
 ) : ViewModel() {
 
-  // a state that holds all displayed parkings slots /selected parkings
-  private val _selectedParkings = MutableStateFlow<List<Parking>>(emptyList())
-  val selectedParkings: StateFlow<List<Parking>> = _selectedParkings
+  /** List of parkings within the designated area */
+  private val _rectParkings = MutableStateFlow<List<Parking>>(emptyList())
+  val rectParkings: StateFlow<List<Parking>> = _rectParkings
+
+  /** List of k (or less) closest parking queried */
```

Comment:
May be interesting to add in comment what use case/screen each StateFlow argument is useful for? Otherwise the comments + StateFlow fields are appropriate

Reply:
I didn't think of that since this dependent on the implementation and up to the choice of the programmer that implements the ViewModel

Reply:
Fair point, in that case the comments are fine as-is!